packages/graphql-typescript-definitions/src/cli.ts

```diff
@@ -47,31 +54,62 @@ const argv = yargs
   })
   .help().argv;
 
+if (typeof argv.cwd === 'string') {
+  process.chdir(argv.cwd);
+}
+
+// convert the graphql globs to a relative path (if provided)
+// this is to deal with an issue with graphql-config and absolute path globbing
+// see: https://github.com/stephen/vscode-graphql/issues/23
+const graphQLFiles = argv._[0] && relative(process.cwd(), argv._[0]);
+
 const builder = new Builder({
-  graphQLFiles: argv._[0],
+  cli: true,
+  graphQLFiles,
   schemaPath: argv.schemaPath,
   schemaTypesPath: argv.schemaTypesPath,
   addTypename: argv.addTypename,
   enumFormat: argv.enumFormat,
 });
 
+const schemas: SchemaBuild[] = [];
+const docs: Build[] = [];
+
 const BUILT = chalk.inverse.bold.green(' BUILT ');
 const ERROR = chalk.inverse.bold.red(' ERROR ');
 
 builder.on('start', () => {
+  docs.length = 0;
   console.log();
 });
 
-builder.on('schema:start', () => {
+builder.on('start:schema', () => {
+  schemas.length = 0;
   console.log();
 });
 
-builder.on('schema:end', () => {
-  console.log(`${BUILT} ${argv.schemaTypesPath}`);
+builder.on('build', (doc) => {
```

Comment:
Can you open a follow-up issue to rename some of these events, they don't really make sense now that we've added the schema ones

Reply:
yea i wanted to maintain compat, but really i'll just fix them all if we are not concerned about compat.